parser/src/main/java/com/microsoft/gctoolkit/parser/G1GCForwardReference.java

```diff
@@ -675,6 +684,10 @@ G1GCPauseEvent buildEvent() {
                         LOGGER.warning("Unrecognized (mostly) Concurrent Cycle Pause Event " + getConcurrentPhase());
                 }
                 return null;
+            case G1GCConcurrentUndoCycle:
+                // gctype is likely incorrectly set in the forward reference. The plan is to specialize the forward
+                // references thus allowing this event to be built here rather than in a specialized method.
```

Comment:
Should this be logged?

Reply:
code will be collapsed into here

Reply:
G1GCForward reference is far too broad which is making it unwieldy.. this will be refactored. The UndoCycle is a new JDK 17 event and will be flushed out in a near future pull request